integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala

```diff
@@ -15,7 +15,5 @@ object Offloader {
 
   val bloopCompileProjectSettings: Seq[Def.Setting[_]] = List()
   val bloopCompileGlobalSettings: Seq[Def.Setting[_]] = List()
-  val bloopCompileConfigSettings: Seq[Def.Setting[_]] = List(
-    BloopKeys.bloopCompile := Keys.compileIncremental.value
```

Comment:
Can you elaborate on why this change is safe?

Reply:
Not 100% sure, I tested it out on a `0.13` project and it worked. `Keys.compileIncremental` can  sometimes be not  there and it will break some projects when they include the Bloop plugin.

I will work on a test case for that though.

Reply:
I expect making this change is fine because we're not using `bloopCompile` directly in `SbtBloop` yet. I removed that to disable compile offloading. It's really surprising to me that `compileIncremental` is not scoped in sbt 0.13, most probably it's a bug.